src/ui/widgets/StandingsWidget/SessionHeader/SessionHeader.module.scss

```
@@ -14,13 +14,25 @@
   overflow: hidden;
   white-space: nowrap;
 }
 
+// Rule between the header's plain-text items — the chips they replaced carried
+// their own borders. Painted from the header's own text color rather than the
+// row border, which is near-black and vanishes against this background.
+.divider {
+  width: 1px;
+  align-self: stretch;
+  margin: sp(xxxs) 0;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)